lambdatimes/src/components/Content/Tab.js

```diff
@@ -8,11 +8,9 @@ const Tab = props => {
   return (
```

Comment:
for the most part you are only going to reference the this keyword when you have class components